build.gradle

```diff
@@ -197,3 +199,16 @@ detekt {
     config = files("detekt.yml")
     input = files("src/")
 }
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                from components.release
+
+                groupId = 'com.nextcloud.android-library'
+                artifactId = 'master'
```

Comment:
Does this lock builds for master or will everything still work for tags and branches?

Reply:
This is only the name of the generated aar file, which is then renamed by jitpack.
```
dependencies {
	        implementation 'com.github.nextcloud:android-library:jitpack-SNAPSHOT'
	}
```
Will work, with this branch.